tests/integration/components/o-s-s/currency-input-test.ts

```diff
@@ -16,6 +16,49 @@ module('Integration | Component | o-s-s/currency-input', function (hooks) {
     assert.dom('.currency-input-container').exists();
   });
 
+  test('it renders currency only', async function (assert) {
+    this.onChange = () => {};
+    this.onlyCurrency = true;
+    await render(
+      hbs`<OSS::CurrencyInput @currency="USD" @onlyCurrency={{this.onlyCurrency}} @onChange={{this.onChange}} />`
```

Comment:
```suggestion
    await render(
      hbs`<OSS::CurrencyInput @currency="USD" @onlyCurrency={{true}} @onChange={{this.onChange}} />`
```
You don't use it in test, set up value directly in the component.